examples/example_pitchbend/main.js

```diff
@@ -0,0 +1,161 @@
+import "./Superpowered.js";
+
+var webaudioManager = null; // The SuperpoweredWebAudio helper class managing Web Audio for us.
+var Superpowered = null; // A Superpowered instance.
+var audioNode = null;    // This example uses one audio node only.
+var content = null;      // The <div> displaying everything.
+var pitchShift = 0;      // The current pitch shift value.
+var currentPath = null;
+var pbPerc = null;
+
+function changePitchShift(e) {
+    // limiting the new pitch shift value
+    let value = parseInt(e.target.value);
+    pitchShift += value;
+    if (pitchShift < -12) pitchShift = -12; else if (pitchShift > 12) pitchShift = 12;
+    // displaying the value
+    document.getElementById('pitch-shift-display').innerText = ' pitch shift: ' + ((pitchShift < 1) ? pitchShift : '+' + pitchShift) + ' ';
+    // sending the new value to the audio node
+    audioNode.sendMessageToAudioScope({ 'pitchShift': pitchShift });
+}
+
+// on change by the rate slider
+function changeRate() {
+    // displaying the new rate
+    let value = document.getElementById('rateSlider').value, text;
+    if (value == 10000) text = 'original tempo';
+    else if (value < 10000) text = '-' + (100 - value / 100).toPrecision(2) + '%';
+    else text = '+' + (value / 100 - 100).toPrecision(2) + '%';
+    document.getElementById('rateDisplay').innerText = text;
+    // sending the new rate to the audio node
+    audioNode.sendMessageToAudioScope({ rate: value });
+}
+
+function changePitchBend(e) {
+    let faster = 1;
+    const value = e.target.value;
+    if (value < 0) {
+        faster = 0;
+    }
+    audioNode.sendMessageToAudioScope({
+        'pitchBend': true,
+        maxPercent: Math.abs(value/100),
+        bendStretch: 0,
+        faster,
```

Comment:
Isn't this a bug? I think it's missing the key "faster".

Something like: `faster: value < 0 ? 0 : 1,`